docs/faq.md

```diff
@@ -33,7 +33,24 @@ library(SingleCellExperiment)
 scpca_sample <- readRDS("SCPCL000000_filtered.rds")
 ```
 
-## What is the difference between a sample and a library?
+## What is the difference between samples and libraries?
+
+A sample ID, labeled as `scpca_sample_id` and indicated by the prefix `SCPCS`, represents a unique tissue that was collected from a participant. 
+
+The library ID, labeled as `scpca_library_id` and indicated by the prefix `SCPCL`, represents a single set of cells from a tissue sample.
+For single-cell or single-nuclei experiments, this will be the result of emulsion and droplet generation using the 10X Genomics workflow, potentially including both RNA-seq, CITE-seq and cell hashing sequencing libraries. 
+For a bulk RNA-seq experiment, this will result in a single sequencing library. 
+
+In most cases, each sample will only have one corresponding single-cell library, and may also have an associated bulk RNA-seq library.
+However, in some cases multiple libraries were created by separate droplet generation and sequencing from the same sample, resulting in more than one library ID being associated with the same sample ID. 
+
+## Why do some samples have missing participant IDs?
+
+The `participant_id` indicates the participant from which a collection of samples was obtained. 
+For example, one participant may have a sample collected both at initial diagnosis and at relapse.
+This would result in two different sample ID's, but the same participant ID. 
+For the most part, each participant only has one corresponding sample that was collected and submitted for sequencing. 
+Because of this, many of the samples do not have a participant ID and only those samples in which multiple unique samples have been obtained from the same participant will a participant ID be present. 
```

Comment:
Separating the second clause and making it more active.
```suggestion
Because of this, many of the samples do not have a separate participant ID. 
Participant IDs are only present for samples that were derived from the same participant as at least one other sample. 
```